api/src/main/java/api/Answer.java

```diff
@@ -24,4 +27,19 @@ public void setAccepted(boolean accepted) {
         this.accepted = accepted;
     }
 
+    public long getQuestionId() {
+        return questionId;
+    }
+
+    public void setQuestionId(long questionId) {
```

Comment:
I think that when we add a new property , we should make sure that the other daos are using it as well. So that we get a more unified api for the frontend .